src/main/scala/mesosphere/marathon/core/instance/Instance.scala

```diff
@@ -1,22 +1,23 @@
 package mesosphere.marathon
 package core.instance
 
+import java.time.Clock
```

Comment:
Is this still required?

Reply:
no. I guess the build will fail as well 🗡